python/ctsm/test/test_unit_fsurdat_modifier.py

```diff
@@ -95,6 +95,17 @@ def test_subgrid_and_idealized_fails(self):
         ):
             read_cfg_option_control(self.modify_fsurdat, self.config, section, self.cfg_path)
 
+    def test_dompft_and_splitcropland_fails(self):
+        """test that dompft and evenly_split_cropland fails gracefully"""
+        section = "modify_fsurdat_basic_options"
+        self.config.set(section, "dom_pft", "1")
+        self.config.set(section, "evenly_split_cropland", "True")
+        with self.assertRaisesRegex(
+            SystemExit,
+            "dom_pft must be UNSET if evenly_split_cropland is True; pick one or the other",
```

Comment:
I recommended narrowing this error check to dom_pft = some crop

Reply:
@samsrabin was this done?

Reply:
It looks like it is, but please verify.

Reply:
Yes, in [5b604ec](https://github.com/ESCOMP/CTSM/pull/2081/commits/5b604ec8886e9968c64998a7ad00567bf76a18a0).